client/src/slices/configuration/configurationSlice.ts

```diff
@@ -0,0 +1,23 @@
+import { createSlice } from '@reduxjs/toolkit'
+
+interface ConfigurationState {
+  useLegacyInvitations: boolean
+}
+
```

Comment:
I think we can merge this with the `ConnectionSlice` as Timo mentioned in #94.

Reply:
Yep. On it